org.qt.photoqt.json

```diff
@@ -0,0 +1,80 @@
+{
+    "id": "org.photoqt.PhotoQt",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.10",
+    "sdk": "org.kde.Sdk",
+    "command": "photoqt",
+    "rename-desktop-file": "photoqt.desktop",
+    "rename-appdata-file": "photoqt.appdata.xml",
+    "rename-icon": "photoqt",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
this seems like too broad a privilege request

Reply:
Hi, I'm the one who updated manifest to be Flathub ready. I'm not sure PhotoQt can work in full sandbox via portal, so there needs to be direct access to the filesystem. Access only to home would be fine? The original manifest had "host" and I thought the app author wanted it to browse outside home as well, so I kept it there.

Reply:
@luspi what's the expected use case here? xdg-pictures and videos? home? viewing random assets from the rest of the system?

Reply:
I think "viewing random assets" is a pretty good description of its use has. It includes of course folders like xdg-pictures and home, but also goes beyond that to anywhere a use might encounter an image of some type...